samples/Infrastructure/Store.fs

```diff
@@ -97,9 +97,9 @@ module Cosmos =
         member x.TipMaxJsonLength =     p.GetResult(TipMaxJsonLength, 30_000)
         member x.QueryMaxItems =        p.GetResult(QueryMaxItems, 10)
 
-    let logContainer (log: ILogger) name (mode, endpoint, db, container) =
```

Comment:
this stuff is just sitting here - will likely make a PR with some tweaks inc this on the road to completing https://github.com/jet/dotnet-templates/pull/131